lua/bufferline/utils.lua

```diff
@@ -24,7 +26,7 @@ local function index(tbl, n)
 end
 
 local function slice(tbl, first, last)
-  if type(tbl) == 'string' then
+  if type(tbl) == TYPE_STRING then
```

Comment:
This is maybe the only case where I don't find advantages to replacing magic-strings with constants.